OnLand/ControlLaptop/templates/_cards/profile.html

```diff
@@ -90,7 +110,7 @@ <h4 class="title">Flight Profile</h4>
                <td>Brake Set A - Max PSI</td>
                <td id="brakeAMaxValue" >500</td>
                <td>PSI</td>
-               <td>Maximium Allowable Pressure in Brake set A</td>
+               <td>Maximum Allowable Pressure in Brake set A</td>
             </tr>
             <tr>
                <td><i id="brakeBStatus" class="fa fa-circle text-success"></i></td>
```

Comment:
We should make the second table on this page its own card. Then we can easily load it on the dashboard page as "last sent config"

Reply:
Can be a task for Future Prs.